### PR TITLE
INT-3302: Update tinymce to version 7

### PR DESCRIPTION
**Pull request by hamza0867**

### CODE REVIEW

File: src/main/ts/Utils.ts [hunk 1]
```diff
@@ -24,6 +24,10 @@ const validEvents = [
   'onClearUndos',
   'onClick',
   'onContextMenu',
+  'onCommentChange',
```
[the remark this answers is not in the document]
Comment (TheSpyder) [reply src/main/ts/Utils.ts:27]: yeah these aren't all new to 7, commentchange was 6.1

**Review by TheSpyder — APPROVED** [review]

File: src/main/ts/Utils.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (hamza0867) [reply src/main/ts/Utils.ts:27]: I added a changelog entry for these events

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+- Default cloud channel to '7'
```
Comment (spocke) [inline CHANGELOG.md:9]: Add categories here I guess this is a Changed category and the other one is a Added

**Review by spocke — APPROVED** [review]

**Review by ArvinJ-H — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (hamza0867) [reply CHANGELOG.md:9]: Right 👍

Comment (hamza0867) [reply CHANGELOG.md:11]: Done!

**Pull request merged**